### PR TITLE
Guido paella recipe

### PR DESCRIPTION
**Pull request by guidoasbun**
Hello Daniel, Please review my code

### CODE REVIEW

File: guido/assets/js/script.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+'use strict'
+
```
Comment (danieladdsn) [inline guido/assets/js/script.js:2]: I the the small, medium, large function you've included.

File: guido/assets/js/script.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    { type: 'seasoning', amountS: 2, amountM: 4, amountL: 6, name: "Pinch Pepper"},
+    { type: 'seasoning', amountS: 1, amountM: 2, amountL: 3, name: "Pinch Saphron"},
+    { type: 'seasoning', amountS: 2, amountM: 4, amountL: 6, name: "tbs Olive Oil"},
+]
+const ingredientsOther = [
+    { type: 'other', amountS: 2, amountM: 4, amountL: 6, name: "lbs Bomba Rice"},
+    { type: 'other', amountS: 2, amountM: 4, amountL: 6, name: "oz Chicken Stock"},
+]
+
+// Function NOT Complete
```
Comment (danieladdsn) [inline guido/assets/js/script.js]: I see you still have not completed it, but I'm looking forward to seeing how it works after you done.

File: guido/index.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
```
Comment (danieladdsn) [inline guido/index.html:1]: Index look 👍🏾

File: guido/index.html [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    <div class="displayLandMeats"></div>
+
+    <h1 class="heading">Instructions</h1>
+    <div class="displayInstructions">Lorem ipsum dolor sit amet consectetur adipisicing elit. Minima nemo eveniet quisquam nihil porro. Quam officiis repudiandae dolorem, odit, tempora numquam ex excepturi fugiat iure vel deserunt enim in cum.</div>
+</div>
+
+
+<script src="./assets/js/script.js"></script>
+</body>
+</html>
```
Comment (danieladdsn) [inline guido/index.html]: so far I enjoyed taking a look at your code, I like some of fun things your adding to your webpage. I didn't find much of anything to critic in your code, so I guess keep going - keep up the good work!!

**Review by danieladdsn — CHANGES_REQUESTED** [review]
so far I enjoyed taking a look at your code, I like some of fun things your adding to your webpage. I didn't find much of anything to critic in your code, so I guess keep going - keep up the good work!!

**Review by danieladdsn — APPROVED** [review]

**Comment by guidoasbun** [comment]
All done!!

**Pull request merged**